scripts/calendar/mabims_islamic_generator.py

```
@@ -0,0 +1,142 @@
+# Observer: Singapore
+OBS_LAT = "1.3521"
+OBS_LON = "103.8198"
+OBS_ELEV = 15.0
+
+
+class MabimsEngine:
+    """
+    Calculates Hijri month starts based on the MABIMS astronomical criteria.
+    """
+
+    def __init__(self):
```

Ruff (0.15.11)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)